packages/schematics/angular/ng-new/schema.json

```diff
@@ -141,7 +141,8 @@
     },
     "zoneless": {
       "description": "Create an initial application that does not utilize `zone.js`.",
-      "type": "boolean"
+      "type": "boolean",
+      "default": true
```

Comment:
The default should not be set here and instead, it should only be set in the application schematic.